### PR TITLE
created components forgot and reset-password and changed http to httpclient

### PR DESCRIPTION
**Pull request by OleksandrHrybovoda**

### CODE REVIEW

File: client/src/app/app_services/department.service.ts [hunk 1]
```diff
@@ -1,7 +1,8 @@
 import {Injectable} from '@angular/core';
 import {Observable} from "rxjs/Rx";
 import {Department} from "../app_models/department";
-import {RequestOptions, Headers, Http} from "@angular/http";
+import { HttpClient } from '@angular/common/http';
+import { httpOptions } from '../app_services/user.service';
 
 const api = 'http://127.0.0.1:3000/api/v1';
```
Comment (dd1mk) [inline client/src/app/app_services/department.service.ts]: import api from src/app/environments/environment

File: client/src/app/home/common/forgot-password/forgot-password.component.html [hunk 2]
```diff
@@ -0,0 +1,21 @@
+<div class="main-wrapper">
+	<div class="card" style="width: 27rem;">
```
Comment (idyac) [inline client/src/app/home/common/forgot-password/forgot-password.component.html]: don't use inline styles

File: client/src/app/home/common/reset-password/reset-password.component.html [hunk 3]
```diff
@@ -0,0 +1,24 @@
+<div class="main-wrapper">
+	<div class="card">
```
Comment (idyac) [inline client/src/app/home/common/reset-password/reset-password.component.html]: format html

File: client/src/app/page/wrapper/wrapper.component.ts [hunk 4]
```diff
@@ -38,7 +41,41 @@ export class WrapperComponent implements OnInit {
...
+            type: {name: item.type, value: this.getTaskType(item.type)},
+            date: this.convertDate(item.date),
+            author: '',
+            content: item.content
+          };
+        });
+      });
+  }
+
+  getStatusValue = (status: string): number => {
```
Comment (idyac) [inline client/src/app/page/wrapper/wrapper.component.ts]: Looks like you should use enum like this:

enum Status {
    High = 0,
    Normal,
    ...
}

File: client/src/app/page/wrapper/wrapper.component.ts [hunk 5]
```diff
@@ -38,7 +41,41 @@ export class WrapperComponent implements OnInit {
...
+    if (status === Status.high) {
+      return 0;
+    }
+    if (status === Status.normal) {
+      return 1;
+    }
+    return 2;
+  }
+
+  getTaskType = (type: string): number => {
```
Comment (idyac) [inline client/src/app/page/wrapper/wrapper.component.ts]: the same as for Status

File: client/src/app/page/wrapper/wrapper.component.ts [hunk 6]
```diff
@@ -38,7 +41,41 @@ export class WrapperComponent implements OnInit {
...
+    if (type === Type.issue) {
+      return 1;
+    }
+    return 0;
+  }
+
+  convertDate(date: number): string {
```
Comment (idyac) [inline client/src/app/page/wrapper/wrapper.component.ts]: put example of convertation to comments:

```javascript
/**
* Example bla-bla-bla
*/
```

**Pull request merged**